main.py

```diff
@@ -140,11 +141,11 @@ def dump_threads(*args):
     Helpful signal handler for debugging threads
     """
 
-    print('\nSIGUSR received, dumping threads\n')
```

Comment:
You forgot the `logger` statement right below this comment! =)

Reply:
done